src/components/projectLinks.tsx

```diff
@@ -0,0 +1,42 @@
+"use client"; // This is a Client Component
+
+import Link from "next/link";
+import { Github, Youtube } from "lucide-react";
+
+type ProjectLinksProps = {
+  githubURL: string | null;
+  ytVideo: string | null;
+};
+
+export function ProjectLinks({ githubURL, ytVideo }: ProjectLinksProps) {
+  return (
+    <div className="flex items-center gap-3">
+      {githubURL && (
+        <Link
```

Comment:
should be a not Link
```suggestion
        <a
```